api/src/main/java/org/pmiops/workbench/profile/ProfileMapper.java

```diff
@@ -0,0 +1,95 @@
+package org.pmiops.workbench.profile;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.db.model.DbStorageEnums;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.institution.InstitutionalAffiliationMapper;
+import org.pmiops.workbench.model.Authority;
+import org.pmiops.workbench.model.Degree;
+import org.pmiops.workbench.model.EmailVerificationStatus;
+import org.pmiops.workbench.model.Profile;
+import org.pmiops.workbench.utils.mappers.CommonMappers;
+
+@Mapper(
+    componentModel = "spring",
+    uses = {
+      AddressMapper.class,
+      CommonMappers.class,
+      DemographicSurveyMapper.class,
+      InstitutionalAffiliationMapper.class,
+      PageVisitMapper.class
+    })
+public interface ProfileMapper {
+  @Mapping(target = "contactEmailFailure", ignore = true) // I don't think we actually use this
```

Comment:
Probably worth having a cleanup ticket here. From a spot check of the fields you've commented here, I agree they should all be  removed at this point.

Reply:
https://precisionmedicineinitiative.atlassian.net/browse/RW-4598